perfschema/common.go

```diff
@@ -80,6 +81,50 @@ func decodeValue(data []byte, cols []*model.ColumnInfo) ([]interface{}, error) {
 	return rvalues, nil
 }
 
+// dumpValue is used for debugging purposes only.
+func dumpValue(funcName string, vals []interface{}) {
+	for _, val := range vals {
+		switch v := val.(type) {
```

Comment:
Sure, this function is used for debugging purposes only, is it possible to use one statement instead of a switch?